integrations/extensions/docs/elasticsearch-install-and-setup/watsonx_discovery_install_and_setup.md

```diff
@@ -0,0 +1,396 @@
+* Create a new namespace for ECK installation
+  ```shell
+  oc create ns ${ES_NAMESPACE}
+  ```
+* Create ECK crds
+  ```shell
+  oc create -f https://download.elastic.co/downloads/eck/2.9.0/crds.yaml
+  ```
+  NOTE: `2.9.0` is the latest supported version by watsonx Discovery
+* Create ECK operator in a different namespace `elastic-system`
+  ```shell
+  oc apply -f https://download.elastic.co/downloads/eck/2.9.0/operator.yaml
+  ```
+* Create a Elasticsearch cluster
+  ```shell
+  cat <<EOF |oc apply -f -
+  apiVersion: elasticsearch.k8s.elastic.co/v1
+  kind: Elasticsearch
+  metadata:
+    name: ${ES_CLUSTER}
+    namespace: ${ES_NAMESPACE}
+  spec:
+    volumeClaimDeletePolicy: DeleteOnScaledownOnly
+    version: 8.11.1
+    nodeSets:
+      - name: default
+        count: 3
+        volumeClaimTemplates:
+          - metadata:
+              name: elasticsearch-data # Do not change this name unless you set up a volume mount for the data path.
+            spec:
+              accessModes:
+                - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 50Gi
+              storageClassName: ${STORAGECLASS}
+        podTemplate:
+          spec:
+            containers:
+              - name: elasticsearch
+                resources:
+                  requests:
+                    memory: 8Gi
+                    cpu: 2
+                  limits:
+                    cpu: 8
+                    memory: 8Gi
+  EOF
+  ```  
+  NOTE: Learn more about configuring compute resources from [here](https://www.elastic.co/guide/en/cloud-on-k8s/2.9/k8s-managing-compute-resources.html).
+  * Monitor Elasticsearch health and creation progress
+    ```shell
+    oc get Elasticsearch -n ${ES_NAMESPACE}
+    ```
+  * Check that the Elasticsearch pods are running 
+    ```shell
+    oc get pods -n ${ES_NAMESPACE}
+    ```
+
+* Create a Kibana instance and associate it with your Elasticsearch cluster
+  ```shell
+  cat <<EOF |oc apply -f -
+  apiVersion: kibana.k8s.elastic.co/v1
+  kind: Kibana
+  metadata:
+    name: ${ES_CLUSTER}
+    namespace: ${ES_NAMESPACE}
+  spec:
+    version: 8.11.1
+    count: 3
+    elasticsearchRef:
+      name: ${ES_CLUSTER}
+    podTemplate:
+      spec:
+        containers:
+        - name: kibana
+          env:
+            - name: NODE_OPTIONS
+              value: "--max-old-space-size=2048"
+          resources:
+            requests:
+              memory: 1Gi
+              cpu: 0.5
+            limits:
+              memory: 2.5Gi
+              cpu: 2
+  EOF
+  ```  
+  NOTE: the container resources are configurable.
+  * Monitor Kibana health and creation progress
+    ```shell
+    oc get kibana -n ${ES_NAMESPACE}
+    ```
+  * A ClusterIP Service is automatically created for Kibana, check it by:
+    ```shell
+    oc get service ${ES_CLUSTER}-kb-http -n ${ES_NAMESPACE} 
+    ```
+
+* Add a enterprise ECK license  
```

Comment:
Small typo:  From the Elasticsearch site, the way they phrase this would be:
**Add an ECK Enterprise license**